apps/web-giddh/src/app/add-company/add-company.component.ts

```diff
@@ -989,6 +1002,15 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
         }
     }
 
+    /**
+     * This will be use for step back
+     *
+     * @memberof AddCompanyComponent
+     */
+    public stepBack(): void {
```

Comment:
Here function name is and its working is  different, 

suggestion  create function for change detection and use it.